packages/lesswrong/components/ea-forum/giving-portal/hooks.ts

```diff
@@ -11,10 +11,10 @@ export const useElectionCandidates = (
       electionName: eaGivingSeason23ElectionName,
       sortBy,
     },
+    limit: 30
```

Comment:
Added this so that all the candidates appear in the pre-voting section (I think the default is 10 and there is no load more in that section)